lib/Db/FaceMapper.php

```diff
@@ -175,6 +198,26 @@ public function deleteUserFaces(string $userId) {
 			->execute();
 	}
 
+	/**
+	 * Unset relation beetwen faces and persons from that user in order to reset clustering
+	 *
+	 * @param string $userId User to drop fo unset relation.
+	 */
+	public function unsetPersonsRelationForUser(string $userId) {
```

Comment:
I might be stupid, but I don't see point of setting this to NULL when you are calling `deleteUserPersons` after you call this? Resetting clustering is done by deleting all entries in `persons` table for a given user, why you need this?

Reply:
Honestly, just to make it more real. The command option resets the clusters... How is the database before clustering? It has the column `person` as NULL. Therefore, I tried to replicate this.